editor/src/clj/editor/bundle_dialog.clj

```diff
@@ -642,6 +642,72 @@
+    (doto (VBox.)
+      (ui/add-style! "settings")
+      (ui/add-style! "html5")
+      (ui/children! [(labeled! "Architectures" architecture-controls)]))))
+
+(defn- load-html5-prefs! [prefs view]
+  (ui/with-controls view [architecture-js-web-check-box architecture-wasm-web-check-box]
+    (ui/value! architecture-js-web-check-box (prefs/get-prefs prefs "bundle-html5-architecture-js-web?" true))
+    (ui/value! architecture-wasm-web-check-box (prefs/get-prefs prefs "bundle-html5-architecture-wasm-web?" false))))
+
+(defn- save-html5-prefs! [prefs view]
+  (ui/with-controls view [architecture-js-web-check-box architecture-wasm-web-check-box]
+    (prefs/set-prefs prefs "bundle-html5-architecture-js-web?" (ui/value architecture-js-web-check-box))
+    (prefs/set-prefs prefs "bundle-html5-architecture-wasm-web?" (ui/value architecture-wasm-web-check-box))))
+
+(defn- get-html5-options [view]
+  (ui/with-controls view [architecture-js-web-check-box architecture-wasm-web-check-box]
+    {:architecture-js-web? (ui/value architecture-js-web-check-box)
+     :architecture-wasm-web? (ui/value architecture-wasm-web-check-box)}))
+
+(defn- set-html5-options! [view {:keys [architecture-js-web? architecture-wasm-web?] :as _options} issues]
+  (ui/with-controls view [architecture-js-web-check-box architecture-wasm-web-check-box ok-button]
+    (doto architecture-js-web-check-box
+      (ui/value! architecture-js-web?)
+      (set-field-status! (:architecture issues)))
+    (doto architecture-wasm-web-check-box
+      (ui/value! architecture-wasm-web?)
+      (set-field-status! (:architecture issues)))
+    (ui/enable! ok-button (nil? (:architecture issues)))))
+
+(defn- get-html5-issues [{:keys [architecture-js-web? architecture-wasm-web?] :as _options}]
+  {:architecture (when-not (or architecture-js-web? architecture-wasm-web?)
+                   [:fatal "At least one architecture must be selected."])})
+
+(deftype HTML5BundleOptionsPresenter [workspace view variant-choices compression-choices]
+  BundleOptionsPresenter
+  (make-views [this owner-window]
+    (let [refresh! (make-presenter-refresher this)]
+      (into [(make-generic-headers "Bundle HTML5 Application")
+             (make-html5-controls refresh! owner-window)]
+             (make-generic-controls refresh! variant-choices compression-choices))))
+  (load-prefs! [_this prefs]
+    (load-generic-prefs! prefs view)
+    (load-html5-prefs! prefs view))
+  (save-prefs! [_this prefs]
+    (save-generic-prefs! prefs view)
+    (save-html5-prefs! prefs view))
+  (get-options [_this]
+    (merge {:platform "js-web"}
+           (get-generic-options view)
+           (get-html5-options view)))
+  (set-options! [_this options]
+    (let [issues (get-html5-issues options)]
+      (set-generic-options! view options workspace)
+      (set-html5-options! view options issues )
+      (set-generic-headers! view issues [:architecture]))))
+
```

Comment:
Now that we've added some options to the HTML5 dialog, do we also need to adjust the dimensions of the `&.html5` selector in `_bundle-dialog.scss`?

Reply:
I don't think so. It looks good:

<img width="397" alt="Screenshot 2023-05-19 at 11 18 09" src="https://github.com/defold/defold/assets/1300688/3e8b40e7-cdbf-438e-b9b8-a6cf4f907b4d">